backend/support/management/commands/check_cog_over_for_year.py

```diff
@@ -16,16 +16,30 @@ class Command(BaseCommand):
     Beware! Deletes any existing rows in SingleAuditChecklist
     """
```

Comment:
This doc comment is incorrect now. Also: does this commend actually delete SAC data?

Reply:
Yes.  This command is for LOCAL environment only.